### PR TITLE
CHANGELOG for 2023.12

### PR DESCRIPTION
**Pull request by tobiasd**
As usual, this will be the last PR merged into the RC branch for the current release.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,9 +1,81 @@
...
+    Fixed a bug that prevented bi-directional delivery of DMs [annando]
+    Fixed a bug in privacy settings for remote-self [annando]
+    Fixed a bug that prevented the selection of photos with captions [Raroun]
+    Fixed a bug in Magic Auth [git-marijus]
+    Improved Bluesky connector core [annando]
+    Improved SSL handling of reverse http proxies [MrPetovan]
+    Improved performance [annando]
+    Improved language detection [annando]
+    Improved display of contact relationships [annando]
+    Impeoved display of notifications [annando, MrPetovan]
```
Comment (annando) [inline CHANGELOG]: Imp**r**oved

File: CHANGELOG [hunk 2]
```diff
@@ -1,9 +1,81 @@
...
+    Improved nodeinfo [MrPetovan]
+    Improved the federation statistics in the admin panel [annando]
+    General code cleanup [AlfredSK, annando, jlamothe, mexon, MrPetovan, nupplaphil, Quix0r, toddy15]
+    Renamed "group" to "circle"
+    Renamed "forum" to "group"
+    Added possibility to hide the server block list [MrPetovan, Quix0r]
+    Added creation of moderation reports [AlfredSK, annando, MrPetovan]
+    Added user controled ignore server functionality [MrPetovan]
+    Added support for Unix domain sockets to Redis [MrPetovan]
+    Added user controled channels for the network stream [annando]
```
Comment (annando) [inline CHANGELOG]: control**l**ed

File: CHANGELOG [hunk 3]
```diff
@@ -1,9 +1,81 @@
...
+    Fixed a bug that could lead to deleting your own account unintended [MrPetovan]
+    Fixed redirections in 2FA settings page [MrPetovan]
+    Fixed a problem with distributing some comments [annando]
+    Fixed a bug that caused public groups not being able to hide members [annando]
+    Fixed RSS/Atom feed of group accounts [MrPetovan]
+    Fixed a bug that prevented bi-directional delivery of DMs [annando]
+    Fixed a bug in privacy settings for remote-self [annando]
+    Fixed a bug that prevented the selection of photos with captions [Raroun]
+    Fixed a bug in Magic Auth [git-marijus]
+    Improved Bluesky connector core [annando]
```
Comment (annando) [inline CHANGELOG:27]: Shouldn't that be part in the addon section?

Comment (MrPetovan) [reply CHANGELOG:27]: You did add some Bluesky provisions in the core itself.

**Review by MrPetovan — APPROVED** [review]

**Review by MrPetovan — APPROVED** [review]

**Review by annando — APPROVED** [review]

**Pull request merged**